{"name": "test_registration_continue", "status": "failed", "statusDetails": {"message": "AssertionError: Cant find element by locator: ('css selector', '#content > form > div > div > input.btn.btn-primary')", "trace": "self = <page_objects.RegisterPage.RegisterPage object at 0x101bb9280>, locator = ('css selector', '#content > form > div > div > input.btn.btn-primary')\n\n    def _verify_element_presence(self, locator: tuple):\n        try:\n>           return WebDriverWait(self.browser, self.browser.t).until(EC.visibility_of_element_located(locator))\n\npage_objects/BasePage.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"644714e13005d89e59c2b8df79116179\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x101ba9e50>, message = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except InvalidSelectorException as e:\n                raise e\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       0   chromedriver                        0x000000010125ed98 chromedriver + 4337048\nE       1   chromedriver                        0x0000000101256e14 chromedriver + 4304404\nE       2   chromedriver                        0x0000000100e83a5c chromedriver + 293468\nE       3   chromedriver                        0x0000000100ec8d50 chromedriver + 576848\nE       4   chromedriver                        0x0000000100f03908 chromedriver + 817416\nE       5   chromedriver                        0x0000000100ebca5c chromedriver + 526940\nE       6   chromedriver                        0x0000000100ebd908 chromedriver + 530696\nE       7   chromedriver                        0x0000000101224de4 chromedriver + 4099556\nE       8   chromedriver                        0x00000001012292a0 chromedriver + 4117152\nE       9   chromedriver                        0x000000010122f52c chromedriver + 4142380\nE       10  chromedriver                        0x0000000101229da0 chromedriver + 4119968\nE       11  chromedriver                        0x0000000101201a74 chromedriver + 3955316\nE       12  chromedriver                        0x0000000101246a48 chromedriver + 4237896\nE       13  chromedriver                        0x0000000101246bc4 chromedriver + 4238276\nE       14  chromedriver                        0x0000000101256a8c chromedriver + 4303500\nE       15  libsystem_pthread.dylib             0x000000018cafffa8 _pthread_start + 148\nE       16  libsystem_pthread.dylib             0x000000018cafada0 thread_start + 8\n\nvenv/lib/python3.9/site-packages/selenium/webdriver/support/wait.py:89: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nbrowser = <selenium.webdriver.remote.webdriver.WebDriver (session=\"644714e13005d89e59c2b8df79116179\")>\n\n    def test_registration_continue(browser):\n        MainPage(browser).go_to_register_page()\n>       continue_registration = RegisterPage(browser).click_continue_registration()\n\ntests/test_opencart.py:102: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npage_objects/RegisterPage.py:17: in click_continue_registration\n    continue_registration = self._element(self.CONTINUE_REGISTRATION)\npage_objects/BasePage.py:27: in _element\n    return self._verify_element_presence(locator)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <page_objects.RegisterPage.RegisterPage object at 0x101bb9280>, locator = ('css selector', '#content > form > div > div > input.btn.btn-primary')\n\n    def _verify_element_presence(self, locator: tuple):\n        try:\n            return WebDriverWait(self.browser, self.browser.t).until(EC.visibility_of_element_located(locator))\n        except TimeoutException:\n>           raise AssertionError(\"Cant find element by locator: {}\".format(locator))\nE           AssertionError: Cant find element by locator: ('css selector', '#content > form > div > div > input.btn.btn-primary')\n\npage_objects/BasePage.py:24: AssertionError"}, "start": 1695470698622, "stop": 1695470704086, "uuid": "2770686c-0e8c-4c35-90df-61a52436b860", "historyId": "037697e2ea7140e4f417868da0ae4ccc", "testCaseId": "037697e2ea7140e4f417868da0ae4ccc", "fullName": "tests.test_opencart#test_registration_continue", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_opencart"}, {"name": "host", "value": "Karinas-MacBook-Air.local"}, {"name": "thread", "value": "63199-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_opencart"}]}